clouddriver-web/src/main/groovy/com/netflix/spinnaker/clouddriver/controllers/AuthorizationSupport.groovy

```diff
@@ -53,6 +53,30 @@ class AuthorizationSupport {
     return true
   }
 
+  /**
+   *   Used for filtering result lists by searching for "account"-like properties.
+   */
+  boolean filterForAccounts(List items) {
+    if (!items) {
+      return true
+    }
+
+    Authentication auth = SecurityContextHolder.context.authentication;
+
+    new ArrayList<>(items).each { Object item ->
+      Map propertySource = item.properties
+      if (item instanceof Map) {
```

Comment:
Seems like there might be more types that would cause trouble here, like `List` or `Array`. If you never call it with those types this is fine. Maybe just add something to the docstring so that doesn't happen in the __future__.